zipkin-autoconfigure/storage-elasticsearch-aws/src/test/java/zipkin/autoconfigure/storage/elasticsearch/aws/ZipkinElasticsearchAwsStorageAutoConfigurationTest.java

```diff
@@ -62,9 +62,8 @@ public void providesAWSSignatureVersion4_whenStorageTypeElasticsearchAndHostsAre
         ZipkinElasticsearchAwsStorageAutoConfiguration.class);
     context.refresh();
 
-    assertThat(context.getBean(OkHttpClient.class).networkInterceptors())
-        .extracting(i -> i.getClass())
-        .contains((Class) AWSSignatureVersion4.class);
+    assertThat(context.getBean(Interceptor.class))
```

Comment:
Can you get a `List<Interceptor>` from the context? Otherwise this looks like an assertion that *may* break in the future when there is more than one `Interceptor` bean, which I thought was the point of switching to the `List`.